index.js

```diff
@@ -0,0 +1,162 @@
+
+        const { owner, repo } = github.context.repo;
+
+        if (body.trim() !== "/rerun") {
+            // unrecognized command
+            core.setOutput("triggered", "false");
+            return;
+        }
+
+        if (!github.context.payload.issue.pull_request) {
+            // not a pull request
+            core.setOutput("triggered", "false");
+            return;
+        }
+
+        if (github.context.payload.issue.state !== "open") {
+            // not open
+            core.setOutput("triggered", "false");
+            return;
+        }
+
+        console.log(`evaluating action for PR#${pull_number}`);
+
+        if (
+            authorAssociation !== "COLLABORATOR" &&
+            authorAssociation !== "OWNER" &&
+            authorAssociation !== "MEMBER" &&
+            authorAssociation !== "CONTRIBUTOR"
+        ) {
+            console.log(
+                `skipped due to authorAssociation: ${authorAssociation}`
+            );
+            core.setOutput("triggered", "false");
+            return;
+        }
+
+        const pull_request = unwrapResult(
+            await octokit.request(
+                "GET /repos/{owner}/{repo}/pulls/{pull_number}",
+                {
+                    owner: github.context.payload.repository.owner.login,
+                    repo: github.context.payload.repository.name,
+                    pull_number: pull_number,
+                }
+            )
+        );
+
+        if (debug) {
+            console.log(
+                `Pull request: ${JSON.stringify(pull_request, undefined, 2)}`
+            );
+        }
+
+        const check_runs = unwrapResult(
+            await octokit.request(
+                "GET /repos/{owner}/{repo}/commits/{ref}/check-runs",
+                {
+                    owner: github.context.payload.repository.owner.login,
+                    repo: github.context.payload.repository.name,
+                    ref: pull_request.head.sha,
+                }
+            )
+        ).check_runs;
+
+        for (let i = 0; i < check_runs.length; i++) {
+            const check = check_runs[i];
+            if (check.conclusion === "failure") {
+                if (debug) {
+                    console.log(
+                        `Check: ${JSON.stringify(check, undefined, 2)}`
+                    );
+                }
+
+                const job = unwrapResult(
+                    await octokit.request(
+                        "GET /repos/{owner}/{repo}/actions/jobs/{job_id}",
+                        {
+                            owner: github.context.payload.repository.owner
+                                .login,
+                            repo: github.context.payload.repository.name,
+                            job_id: check.id,
+                        }
+                    )
+                );
+
+                if (debug) {
+                    console.log(`Job: ${JSON.stringify(job, undefined, 2)}`);
```

Comment:
The console can format objects for us.
```suggestion
                    console.log('Job:', job);
```